test/python/circuit/library/test_mcmt.py

```diff
@@ -264,6 +265,17 @@ def test_invalid_base_gate_width(self):
                 with self.assertRaises(ValueError):
                     _ = MCMTGate(gate, 10, 2)
 
+    def test_gate_with_parameters_vchain(self):
```

Comment:
would you like to add a test for `CRYGate` as well?

Reply:
`synth_mcmt_vchain` doesn't support already controlled gates 🙂 Maybe actually that's a check we should add... `CRYGate` could be passed into `MCMTGate`, but then it will just be reduced to `RYGate`, so that path is already covered 🙂

Reply:
Additional check added in f656fa4 🙂